root/programs/C#/Frameworks/Infrastructure/Framework/Transmission/CallController.cs

```diff
@@ -1253,7 +847,233 @@ public object Invoke(string serviceName, object parameterValue)
+                // 実行アカウントでのWindows認証の有効・無効（Default：false）。                    
+                proxy.UseDefaultCredentials = true; // 有効
+
+                #region Proxyのクライアント認証のセキュリティ資格情報
+
+                // Proxyのセキュリティ資格情報
+                if (this._nwcProxy == null)
+                {
+                    // ユーザ指定：なし
+
+                    if (!props.ContainsKey(FxLiteral.TRANSMISSION_HTTP_PROP_PROXY_USER_NAME))// Dic化でnullチェック変更
+                    {
+                        // XML定義：キーが無い
+                    }
+                    else
+                    {
+                        if (props[FxLiteral.TRANSMISSION_HTTP_PROP_PROXY_USER_NAME] == null
+                            || (string)props[FxLiteral.TRANSMISSION_HTTP_PROP_PROXY_USER_NAME] == "")
+                        {
+                            // XML定義：null or 空文字列
+                        }
+                        else
+                        {
+                            // XML定義：あり
+
+                            // Proxyのセキュリティ資格情報を生成する。
+                            NetworkCredential nwcProxy = new NetworkCredential();
+
+                            nwcProxy.UserName = (string)props[FxLiteral.TRANSMISSION_HTTP_PROP_PROXY_USER_NAME];
+                            nwcProxy.Password = (string)props[FxLiteral.TRANSMISSION_HTTP_PROP_PROXY_PASSWORD];
+
+                            // 省略可能に変更した。
+                            if (props.ContainsKey(FxLiteral.TRANSMISSION_HTTP_PROP_PROXY_DOMAIN))
+                            {
+                                nwcProxy.Domain = (string)props[FxLiteral.TRANSMISSION_HTTP_PROP_PROXY_DOMAIN];
+                            }
+
+                            // Proxyのセキュリティ資格情報を設定する（XML定義）。
+                            proxy.Credentials = nwcProxy;
+                        }
+                    }
+                }
+                else
+                {
+                    // ユーザ指定：あり
+
+                    // Proxyのセキュリティ資格情報を設定する（ユーザ指定）。
+                    proxy.Credentials = this._nwcProxy;
+                }
+
+                #endregion
+            }
+
+            #endregion
+
+            return proxy;
+        }
+
+        /// <summary>CreateCredentials</summary>
+        /// <param name="props">Dictionary(string, string)</param>
+        /// <returns>NetworkCredential</returns>
+        private NetworkCredential CreateCredentials(Dictionary<string, string> props)
+        {
+            NetworkCredential nwcWAS = null;
+
+            // WASのセキュリティ資格情報
+            if (this._nwcWAS == null)
+            {
+                // ユーザ指定：なし
+
+                if (!props.ContainsKey(FxLiteral.TRANSMISSION_HTTP_PROP_USER_NAME))// Dic化でnullチェック変更
+                {
+                    // XML定義：キーが無い
+                }
+                else
+                {
+                    if (props[FxLiteral.TRANSMISSION_HTTP_PROP_USER_NAME] == null
+                        || (string)props[FxLiteral.TRANSMISSION_HTTP_PROP_USER_NAME] == "")
```

Comment:
@daisukenishino 

Similar to [this](https://github.com/OpenTouryoProject/OpenTouryo/pull/253/files#r134915067).